src/track/track_gps_l1ca.c

```diff
@@ -238,7 +238,53 @@ static void tracker_gps_l1ca_update(const tracker_channel_info_t *channel_info,
   corr_t* cs = data->cs;
 
   /* Update C/N0 estimate */
-  common_data->cn0 = cn0_est(&data->cn0_est, cs[1].I/data->int_ms, cs[1].Q/data->int_ms);
+  {
+    /* Pre-computed C/N0 estimator and filter parameters. The parameters are
+     * computed using equivalent of cn0_est_compute_params() function for
+     * integration periods of 1, 2, 4, 5, 10 and 20ms and cut-off frequency
+     * of 0.1 Hz.
+     */
+    static const cn0_est_params_t pre_computed[] = {
+      {3.0000000e+01f, 9.8652211e-08f, -1.9991114e+00f, 9.9911182e-01f},
+      {2.6989700e+01f, 3.9443364e-07f, -1.9982228e+00f, 9.9822443e-01f},
+      {2.3979400e+01f, 1.5763345e-06f, -1.9964457e+00f, 9.9645200e-01f},
+      {2.3010300e+01f, 2.4619300e-06f, -1.9955571e+00f, 9.9556697e-01f},
+      {2.0000000e+01f, 9.8259168e-06f, -1.9911143e+00f, 9.9115360e-01f},
+      {1.6989700e+01f, 3.9130205e-05f, -1.9822289e+00f, 9.8238545e-01f}
+    };
+
+    cn0_est_params_t params;
+    const cn0_est_params_t *pparams = NULL;
+    switch (data->int_ms) {
+    case 1:
+      pparams = &pre_computed[0];
+      break;
```

Comment:
I've double checked the Linux coding style. It is not there. Probably this should be added to the internal guidelines.